Dockerfile

```diff
@@ -12,10 +12,10 @@ RUN git clone https://github.com/paritytech/polkadot-sdk.git && cd polkadot-sdk
 # Delete below three lines when polka-storage repo will become public
 WORKDIR /usr/src/polka-storage
 COPY . .
-RUN cargo build --release --features runtime-benchmarks
+RUN cargo build --release
 
-# Uncomment when polka-storage repo will become public
-#RUN git clone https://github.com/eigerco/polka-storage.git && cd polka-storage && cargo build --release --features runtime-benchmarks
+# Uncomment when polka-storage repo will become public and available for cloning for everyone
+#RUN git clone https://github.com/eigerco/polkadot-native-storage polka-storage && cd polka-storage && cargo build --release --features runtime-benchmarks
 
```

Comment:
Is there any reason to rename the repo to `polka-storage`?  I would prefer not to, but maybe you have reasons?  E.g., do many scripts depend on the name `polka-storage`?

Reply:
The one and only reason is maintainability.
It should be easier to keep the main repo and the public one aligned when there will be only one line to change (and the rest of the script can use `polka-storage`). But if it's somehow unwanted - no problem, I'll change it 👍

Reply:
Understood.  Then let's keep it.